packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -108,6 +110,7 @@ export const handler = async (
   const { now, log } = perfLogger(manifest.logLambdaExecutionTimes);
 
   const tHandlerBegin = now();
+  console.log("we are in v1 handler");
```

Comment:
Remove, seems for debugging purposes? (yes the platform client only applies to v2 handlers which is used for lambda and the generic handlers)

Reply:
Well the story behind this is longer.. i wrote in my test config file `useV2Handlers` instead `useV2Handler` and that bugged me for about an hour :D I'll remove it...